client/src/components/Header.js

```diff
@@ -1,31 +1,31 @@
-import React from "react";
-import "../styles/header.css";
-import { Link } from "react-router-dom";
-import Search from "./Search";
+import React from 'react';
+import '../styles/header.css';
+import { Link } from 'react-router-dom';
+import Search from './Search';
 
 const Header = () => {
-	return (
-		<>
-			<div className="tac">
-				<h1>Event Manager</h1>
-				<Search />
-				<Navbar />
-			</div>
-		</>
-	)
-}
-export default Header
+  return (
+    <>
+      <div className="tac">
+        <h1 className="header-header">Event Manager</h1>
```

Comment:
Why does the header have a header? Read as, we need a better class name. Or maybe this could be applied to `.header h1`? Again, I'm not a frontend engineer, so I'm not sure what best practices are.

Reply:
Changed to className="header-title"